scripts/create_newcase

```diff
@@ -12,7 +12,7 @@ def parse_command_line(args, cimeroot, description):
 ###############################################################################
     help_str = \
 """
-%s --case [CASE] --compset [COMPSET] --res [GRID] [--mach ...] [--compiler ...]
+%s --case [CASE] --compset [COMPSET] --res [GRID] [--machine ...] [--compiler ...]
```

Comment:
I think we want to keep --mach for backwards compatibility.  Just don't put it in the help.  Also this should be a separate PR.